shared/oae/api/oae.core.js

```diff
@@ -84,7 +85,8 @@ define([
         'markdown',
         'sockjs',
         'tinycon',
-        'underscore'
+        'underscore',
+        'cookies'
```

Comment:
I'd still be happier if we had the rest of the external libs here too... (Also, I think these should be alphabetical.)

Reply:
Alphabetical order 🆗 

What other external libs? Are we missing some?

Reply:
> What other external libs? Are we missing some?

Ignore me, I thought we were in the bootstrap file. 🤦‍♀️